mslib/msui/qt5/ui_add_project_dialog.py

```diff
@@ -2,43 +2,45 @@
 
 # Form implementation generated from reading ui file 'mslib/msui/ui/ui_add_project.ui'
 #
-# Created by: PyQt5 UI code generator 5.9.2
+# Created by: PyQt5 UI code generator 5.12.3
```

Comment:
this is also the version of qt we currently can't have on a fresh installation on linux, I check this tom. with the old version. Anyway after we changed the channel order we will all have 5.12.9